app/models/notification.rb

```diff
@@ -0,0 +1,14 @@
+class Notification < ApplicationRecord
+  acts_as_readable
+
+  scope :active, -> { where("start_date <= ?", Time.zone.now).where("end_date >= ?", Time.zone.now) }
```

Comment:
would something like this work? `where("start_date <= ? AND end_date >= ?", Time.zone.now, Time.zone.now)`

Reply:
yeah, like it :D